### ISSUE TITLE
Make a logo for the plugin

### ISSUE THREAD

**Issue by AObuchow** [AObuchow/Eclipse-Modern-Dark-Theme#67]
I believe for the Eclipse marketplace, a plugin logo/icon is required.

This bug is to discuss logo ideas & prototypes.

Some ideas:
- Abbreviate “Modern Dark Theme”: MDT, MDT4e, MDT4E
- Use the Eclipse “e” logo in the “o” of “modern”

**Comment by AObuchow** [comment]
Abbreviating to MDT might be a bad idea for branding as [Eclipse Model Development Tools ](https://www.eclipse.org/modeling/mdt/) already exists.

Too bad, would have been a nice abbreviation

**Comment by AObuchow** [comment]
Some quick ideas, using the main colors from the theme:
![image](https://user-images.githubusercontent.com/10300119/85088528-57948b00-b1ae-11ea-8756-77eef0ca84dc.png)

![image](https://user-images.githubusercontent.com/10300119/85088538-5bc0a880-b1ae-11ea-8863-fb46e5f3dfbd.png)

If anyone has any font suggestions, please let me know :)

**Comment by AObuchow** [comment]
New idea: use the main design elements (square tabs, view outline) as a theme for the logo.

![image](https://user-images.githubusercontent.com/10300119/85098734-7eac8600-b1c9-11ea-8e99-77c041686e66.png)

![image](https://user-images.githubusercontent.com/10300119/85098745-8409d080-b1c9-11ea-9a5c-2cc18e7336bf.png)

![image](https://user-images.githubusercontent.com/10300119/85098757-8c620b80-b1c9-11ea-9a79-32baab2fe555.png)

![image](https://user-images.githubusercontent.com/10300119/85098761-91bf5600-b1c9-11ea-8cb7-12425f8d868d.png)

Maybe the "Dark Theme" text should be a monospaced font, to resemble a text editor font?

**Comment by AObuchow** [comment]
Slightly more symmetric version of 6:

7. 
![image](https://user-images.githubusercontent.com/10300119/85099536-a69ce900-b1cb-11ea-9f6b-e8e277688645.png)

**Comment by AObuchow** [comment]
The final logo will be made in Krita and it’s source file should be included in the repo (as long as it’s not too big of a file).

**Comment by AObuchow** [comment]
An idea that came up was to use the Eclipse logo as if it were an editor tab icon, similar to how Eclipses editor tabs have icons to the left of their text.

However this would require permission from the Eclipse foundations legal department https://www.eclipse.org/legal/logo_guidelines.php

**Comment by ingomohr** [comment]
Maybe we can start w/ something like no.7 - as I think a logo is required for the Eclipse Market Place entry.
From there, we maybe can build up later.

(just thinking...) :)

**Comment by AObuchow** [comment]
@ingomohr agreed, so far I think # 7 is the best :) unless someone (myself included) can come up with something better, I'll go with it and improve upon it later

**Comment by AObuchow** [comment]
8. ![image](https://user-images.githubusercontent.com/10300119/85128233-e0d3ae00-b1fe-11ea-95bc-8ef0a371c27c.png)

9. ![image](https://user-images.githubusercontent.com/10300119/85128254-e7622580-b1fe-11ea-8a45-bec427a8ea51.png)

**Comment by ingomohr** [comment]
Hm, I like the 7 better, I think.
_But_ thinks like these always lie in the...
![images](https://user-images.githubusercontent.com/2838592/85128599-e7c7e400-b231-11ea-8eba-42b832b472b7.jpeg)
;)

**Comment by AObuchow** [comment]
@ingomohr hahahahaha ;) I think I'm also +1 for # 7 still as it's more symmetric

**Comment by AObuchow** [comment]
Based on feedback from friends, family and [twitter](https://twitter.com/AObuchow/status/1273945799928492037?s=20), I've decided to go with design # 7 for now.

As a result of [the decision to rename the plugin](https://github.com/AObuchow/Eclipse-Modern-Dark-Theme/issues/69#issuecomment-647867505) to _Eclipse Spectrum Theme_, the logo will use the new name.

![image](https://user-images.githubusercontent.com/10300119/85359052-42d43200-b4e3-11ea-8547-e84abed55ae9.png)

I decided to use a monospaced font for "Theme" since it's supposed to represent a text editor.

I'm going to submit a PR that will contain the Krita source file for the logo as well as the .png
